src/components/SignUp/OrganizationInformation.tsx

```diff
@@ -7,7 +7,9 @@ import {
   isValidPhoneNumber, isValidUSState, isValidZipCode, isValidOrgName,
   isValidEIN,
 } from '../../lib/Validations/Validations';
-import CompleteSignupFlow from './CompleteSignupFlow';
+// import CompleteSignupFlow from './CompleteSignupFlow';
+import 'jquery-mask-plugin';
+import $ from "jquery";
```

Comment:
jquery is an okay temporary fix, but lets try to find another way to do it. This is bc react is supposed to be a replacement of jquery. I would even be fine if you wrote a simple function that detects the dash and adds it in.